includes/feed/cache/class-cache.php

```diff
@@ -59,7 +59,7 @@ class Cache {
 	 */
 	public function __construct( string $url ) {
 		$this->url  = html_entity_decode( $url );
-		$this->hash = wp_hash( $this->url );
+		$this->hash = wp_hash( $url );
```

Comment:
this should go back to using `$this->url` because we want `html_entity_decode` to be applied.